src/network/network_server.cpp

```diff
@@ -1813,12 +1791,63 @@ void NetworkServer_Tick(bool send_frame)
 	}
 }
 
+/** Helper function to restart the map. */
+static void RestartMapHelper()
```

Comment:
`Helper` feels a bit awkward .. why not just call it for what it is? `NetworkRestartMap` or something?